app/src/main/res/layout/changes_item.xml

```diff
@@ -259,7 +259,7 @@
                     android:layout_height="wrap_content"
                     android:layout_marginTop="3dp"
                     android:layout_below="@id/statuses_block"
-                    android:visibility="@{!safeUnbox(model.isWorkInProgress) &amp;&amp; !model.mergeable &amp;&amp; !model.status.equals(ChangeStatus.MERGED) &amp;&amp; !model.status.equals(ChangeStatus.ABANDONED) ? View.VISIBLE : View.GONE}"
+                    android:visibility="@{!safeUnbox(model.isWorkInProgress) &amp;&amp; Boolean.FALSE.equals(model.mergeable) &amp;&amp; !model.status.equals(ChangeStatus.MERGED) &amp;&amp; !model.status.equals(ChangeStatus.ABANDONED) ? View.VISIBLE : View.GONE}"
```

Comment:
It'd likely be very slow to fetch mergeable to each change one by one, so I just dropped `oolean.FALSE.equals(model.mergeable)` here.